src/stcal/alignment/util.py

```diff
@@ -14,32 +20,39 @@
 from astropy.utils.misc import isiterable
 from gwcs.wcstools import wcs_from_fiducial
 
-if TYPE_CHECKING:
-    from asdf import AsdfFile
-
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
-
 __all__ = [
     "compute_scale",
     "compute_fiducial",
     "calc_rotation_matrix",
+    "update_s_region_imaging",
+    "update_s_region_keyword",
     "wcs_from_footprints",
     "reproject",
+    "Wcsinfo"
 ]
 
 
-class SupportsDataWithWcs(Protocol):
-    _asdf: AsdfFile
+@runtime_checkable
+class Wcsinfo(Protocol):
```

Comment:
```suggestion
class Wcsinfo(Protocol):
```
I don't see any uses of this outside of the type hints. The `runtime_checkable` can be removed.